scipy/stats/stats.py

```diff
@@ -3707,7 +3709,8 @@ def _ttest_ind_from_stats(mean1, mean2, denom, df):
 def _unequal_var_ttest_denom(v1, n1, v2, n2):
     vn1 = v1 / n1
     vn2 = v2 / n2
-    df = ((vn1 + vn2)**2) / ((vn1**2) / (n1 - 1) + (vn2**2) / (n2 - 1))
+    with np.errstate(divide='ignore', invalid='ignore'):
+        df = ((vn1 + vn2)**2) / ((vn1**2) / (n1 - 1) + (vn2**2) / (n2 - 1))
```

Comment:
Optional: while you're touching this line, it'd be nice to remove extra brackets: 
```
(vn1 + vn2)**2 / (vn1**2 / (n1 - 1) + vn2**2 / (n2 - 1))
```

looks a bit more readable.